src/components/template-gallery/TemplateGalleryItem.tsx

```
@@ -0,0 +1,66 @@
+      className={cn(
+        "group relative cursor-pointer rounded-lg overflow-hidden",
+        "border border-border/30 hover:border-border/60",
+        "transition-all duration-300 ease-out",
+        "hover:shadow-lg hover:scale-[1.02]",
+        "opacity-0 animate-fade-in-up"
+      )}
+      style={{
+        animationDelay: `${index * 50}ms`,
+      }}
+      onClick={() => onClick(template)}
+    >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)